app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  add_flash_types :success, :info, :warning, :danger
```

Comment:
Is this so you can add in custom `flash` types, vs. the standard `[:notice, :error, :alert]`?